src/pd/client.rs

```diff
@@ -25,6 +25,16 @@ use crate::{
 const CQ_COUNT: usize = 1;
 const CLIENT_PREFIX: &str = "tikv-client";
 
+#[macro_export]
+macro_rules! group_by_range {
```

Comment:
I doubt whether this macro is useful. It looks no different from just calling `group_ranges_by_region` or `group_keys_by_region`.
I guess @nrc hopes to make `group_ranges_by_region` and `group_keys_by_region` share some code?

Reply:
I've made some efforts on this, but it seems not able to merge the group function of key-like data and range-like data together.

Reply:
Ok, that can be a bit difficult.
But then I think this macro is unnecessary. What do you think @nrc ?

Reply:
Yeah, I'm happy either way (thanks for trying @you06 !)

Reply:
The macro has been removed.